pkg/gadgets/snapshot/process/types/process-collector_test.go

```diff
@@ -0,0 +1,196 @@
+// Copyright 2022 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package types
+
+import (
+	"bufio"
+	"fmt"
+	"strings"
+	"testing"
+)
+
+type stsTestCase struct {
+	val      uint64
+	valMax   uint64
+	width    uint64
+	expected string
+}
+
+func TestCreateTree(t *testing.T) {
+	t.Parallel()
+
+	events := []*Event{
+		{
+			Pid: 42,
+		},
+		{
+			Pid:       43,
+			ParentPid: 42,
+		},
+		{
+			Pid:       44,
+			ParentPid: 42,
+		},
+		{
+			Pid:       45,
+			ParentPid: 42,
+		},
+		{
+			Pid:       46,
+			ParentPid: 43,
+		},
+	}
+
+	tree, err := createTree(events)
+	if err != nil {
+		t.Fatalf("fail to create tree: %v", err)
+	}
+
+	expectedPid := 42
+	if tree.process.Pid != expectedPid {
+		t.Fatalf("root PID is wrong, expected %d got %d", expectedPid, tree.process.Pid)
+	}
```

Comment:
I know you created this PR before we started using `github.com/stretchr/testify`, however it could improve these tests a lot. 

`require.Equal(t, 42, tree.process.Pid, "root PID is wrong")`